extensions/amp-3d-gltf/amp-3d-gltf.md

```diff
@@ -0,0 +1,90 @@
+<!--
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="`amp-3d-gltf`"></a> `amp-3d-gltf`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Displays GL Transmission Format (gITF) 3D models.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Experimental</td>
```

Comment:
I changed the availability information, as it typically specifies one of the following: experimental, in development.

If this component is not experimental when released, please remove this complete table row.

Reply:
won't be experimental,  but would take a few weeks before validation rules make it to prod. Maybe keep it until then?